codegen/module_system.go

```diff
@@ -255,7 +255,7 @@ type HTTPClientGenerator struct {
 // path (relative to the target build directory) to file bytes.
```

Comment:
The comments of all the `Generate` method need to be updated to reflect the return value change.